src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -2177,18 +2168,21 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		// Check for editor transfer
 		else if (this.editorTransfer.hasData(DraggedEditorIdentifier.prototype)) {
 			const data = this.editorTransfer.getData(DraggedEditorIdentifier.prototype);
-			if (Array.isArray(data)) {
+			if (Array.isArray(data) && data.length > 0) {
```

Comment:
The logic here seems to have changed from allowing a single-editor drop to multi-editor drop. But I am not fully understanding the implementation: we now loop over each editor and call `moveEditor` or `copyEditor`, but isn't that inefficient? We have methods to move/copy multiple editors at once, so I would have assumed that to be used.

Reply:
This is used beacuse it is not possible to keep the order of the editors using `moveEditors` and `copyEditors` because of the indices changing. The Open Editors view has the same challenge and has always used the approach.